# This patch fixes the openpgp tests.
# The version contains the string "MacGPG2".

--- a/tests/openpgp/version.scm
+++ b/tests/openpgp/version.scm	
@@ -22,4 +22,4 @@
 
 (info "Printing the GPG version")
 (assert (string-contains? (call-check `(,@GPG --version))
-			  "gpg (GnuPG) 2."))
+			  "gpg (GnuPG/MacGPG2) 2."))
